### ISSUE TITLE
Roadmap to 1.0?

### ISSUE THREAD

**Issue by Torwori** [sindresorhus/xo#181]
Hi,

do you have a roadmap to 1.0? It would be nice to get the benefits of semver-minor and semver-patch updates. :smile:

**Comment by sindresorhus** [comment]
Semver is difficult for linters, especially this one, which comes with default config. Any new rule, for example, would have to be a new major release, as it could potentially break your CI.

**Comment by Torwori** [comment]
Is there any downside to that, other than large major version numbers?

Love what you've made btw. :+1:

**Comment by sindresorhus** [comment]
Not really. Just seems some people get annoyed by regular major versions, but not regular pre-1.0.0 minor versions, even though they have the same semantics. I don't think XO is 1.0.0 material quite yet though. There are some important things I want to fix first and improve performance. I'll keep this issue open so I don't forget.

**Comment by jodaka** [comment]
@sindresorhus  do you it's ready for 1.0.0 4 years later?

**Comment by sindresorhus** [comment]
Yeah, I guess. I can do a major version for the next release.

**Comment by fregante** [comment]
v1 is here ☺️
